homeassistant/const.py

```diff
@@ -128,6 +128,7 @@
 CONF_OFFSET = "offset"
 CONF_OPTIMISTIC = "optimistic"
 CONF_PACKAGES = "packages"
+CONF_PARAMS = "params"
```

Comment:
Should be `parameters`.

```suggestion
CONF_PARAMETERS = "parameters"
```

Reply:
Naming is directly taken from `httpx`. `requests` uses the same naming.

Reply:
@fabaff